applications/crossbar/src/api_util.erl

```diff
@@ -868,17 +868,24 @@ ensure_content_type(CT) -> CT.
 %% (the final module in the chain) accepts this verb parameter pair.
 %% @end
 %%--------------------------------------------------------------------
--spec does_resource_exist(cb_context:context()) -> boolean().
--spec does_resource_exist(cb_context:context(), list()) -> boolean().
+-spec does_resource_exist(cb_context:context()) -> {boolean(), cb_context:context()}.
+-spec does_resource_exist(cb_context:context(), list()) -> {boolean(), cb_context:context()}.
 does_resource_exist(Context) ->
     does_resource_exist(Context, cb_context:req_nouns(Context)).
 
 does_resource_exist(Context, [{Mod, Params}|_]) ->
     Event = api_util:create_event_name(Context, <<"resource_exists.", Mod/binary>>),
-    Responses = crossbar_bindings:map(Event, Params),
-    crossbar_bindings:any(Responses) and 'true';
-does_resource_exist(_Context, _ReqNouns) ->
-    'false'.
+    Responses = crossbar_bindings:map(Event, [Context, Params]),
+    SuccessResponses = [X || {'true', X} <- Responses],
```

Comment:
I think I'd like to use (`crossbar_bindings:succeeded/1`)[https://github.com/2600hz/kazoo/blob/master/applications/crossbar/src/crossbar_bindings.erl#L97] here instead.